File: app/src/main/java/ifteam/affogatoman/fontgen/LicenseActivity.java
package ifteam.affogatoman.fontgen;

import android.app.Activity;
import android.os.Bundle;
import defpackage.LogCatBroadcaster;

public class LicenseActivity extends Activity {
    @Override
    public void onCreate(Bundle bundle) {
        Bundle bundle2 = bundle;
        LogCatBroadcaster.start(this);
        super.onCreate(bundle2);
        setContentView(R.layout.license);
    }

    public LicenseActivity() {
        Activity activity = this;
    }
}

File: app/src/main/java/ifteam/affogatoman/fontgen/LicenseActivity.kt
package ifteam.affogatoman.fontgen

import android.app.Activity
import android.os.Bundle

class LicenseActivity : Activity() {
    public override fun onCreate(bundle: Bundle?) {
        super.onCreate(bundle)
        setContentView(R.layout.license)
    }

    init {
        val activity: Activity = this
    }
}